src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -1,14 +1,20 @@
 package mate.academy.service;
 
+import mate.academy.Main;
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger(Main.class);
```

Comment:
The logger is currently set up with `Main.class`, which is not appropriate for this class. It should be `AuthenticationServiceImpl.class` to correctly reflect the class where the logger is used.